### DISCUSSION TITLE
神隐漫画相关

### DISCUSSION THREAD

**Discussion by sfrdom** [xiaoyaocz/flutter_dmzj#116] [General]
老版本还是可以在搜索里找到神隐的漫画，现在新版本好像搜不到了。然后神隐漫画可以让它在我的订阅里显示吗？现在只能在首页我的订阅那三个漫画显示一下，其他漫画一更新就会被刷掉了

**Comment by oubeichen** [comment]
设置中打开“使用Web接口搜索漫画”

**Reply by sfrdom** [reply]
好的，谢谢。那实现我的订阅里面显示神隐漫画是比较困难的吗(*꒦ິ⌓꒦ີ)

**Reply by xiaoyaocz** [reply]
下版本会开发一个"本地收藏"功能，可以把神隐漫画添加到"本地收藏"
